spec/models/task.rb

```diff
@@ -12,4 +12,21 @@
     task = Task.new(title: '成功テスト', description: '成功テスト')
     expect(task).to be_valid
   end
+  describe '検索機能' do
+    context 'scopeメソッドで検索をした場合' do
+      before do
+        Task.create(title: "task", description: "sample_task", status: "着手中")
+        Task.create(title: "sample", description: "sample_sample")
+      end
```

Comment:
こちらも同義となります。
```suggestion
      let!(:task){ Task.create(title: "task", description: "sample_task", status: "着手中") }
      let!(:task){ Task.create(title: "sample", description: "sample_sample") }
```

letで記載する場合はcontextではなく、describeの直後に定義する必要があるかもしれません。
また、 let! と記載すると即時実行され、let で記載すると設定した 引数のシンボルが実行された場合に対応する処理を実行することになります。

`let(:task) { FactoryBot.create(:task) }`と記載すると task と記載することでFactoryBotの作成処理が行われてtaskというローカル変数で扱えるようになります。

Reply:
こちら未修正です。